docs/js-core-3/week-3/lesson.mdx

```diff
@@ -287,319 +285,168 @@ function six() {
 
-```js
-class Person {
-  constructor() {
-    this.name = "Mike";
-  }
-
-  sayHi() {
-    console.log(`Hi, ${this.name}`);
-  }
-}
+Which of the following lines of code can be put on line 5?
 
-const person = new Person();
-const sayHi = person.sayHi;
+- A) let sum, multiply = sumAndMultiply(2, 3);
+- B) let [sumAndMultiply] = sumAndMultiply(2, 3);
+- C) let [sum, multiply] = sumAndMultiply(a, b);
+- D) let [sum, multiply] = sumAndMultiply(2, 3);
 
-sayHi(); // TypeError: Cannot read property 'name' of undefined
 ```
 
-In above example, we are executing `sayHi` as a function, not method. As a result we no longer have access to the object the method belongs to. That results in the value of `this` being lost and instead of being `person` it is now `undefined`.
-
-That can be a problem. Fortunately we can fix it
-
-### arrow functions to the rescue
-
-Latest versions of JavaScript allow us to declare methods in classes using arrow functions. When an arrow function is used, the value of `this` inside it is bound to the object it belongs to. That allows us to call methods as functions without losing the value of `this` inside it.
+</TabItem>
+<TabItem value="solution">
 
 ```js
-class Person {
-  constructor() {
-    this.name = "Mike";
-  }
-
-  sayHi = () => {
-    console.log(`Hi, ${this.name}`);
-  };
-}
-
-const person = new Person();
-const sayHi = person.sayHi;
-
-sayHi(); // Hi, Mike
+D) let [sum, multiply] = sumAndMultiply(2, 3);
 ```
 
-This behaviour will become important in the near future, when we will need to start passing methods to be called other parts of our code.
-
-### Exercises
+</TabItem>
+</Tabs>
 
-In break out groups solve the following exercises. Only run code to verify answers. Try to understand what each line of code is doing and the effect it has.
+#### Exercise (4)
 
-```js
-// Exercise 1
-class Car {
-  goFaster() {
-    this.speed = 100; // what is the value of this code below is run
-  }
-}
+Complete all of the exercises found [here](https://repl.it/@ChrisOwen101/ArrayDestructing)
 
-const ferrari = new Car();
-ferrari.goFaster();
+### Resources
 
-// Exercise 2
-class Car {
-  goFaster() {
-    this.speed = 100; // what is the value of this code below is run
-  }
-}
+Here are some useful resources to help you understand this material
 
-const honda = new Car();
-const quicker = honda.goFaster;
-
-quicker();
-
-// Exercise 3
-
-// How can we fix the bug above? Why do have a problem and how does your solution fix it?
-```
+- [FreeCodeCamp's Intro To Array Destructuring](https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/)
+- [Why Is Array/Object Destructuring So Useful And How To Use It (Video)](https://www.youtube.com/watch?v=NIq3qLaHCIs)
+- [The Most In-Depth Yet Understandable ES6 Destructuring Tutorial](https://untangled.io/in-depth-es6-destructuring-with-assembled-avengers)
```

Comment:
I'd suggest Wes Bos' blog post too: https://github.com/wesbos/es6-articles/blob/master/19%20-%20Destructing%20Arrays.md (he also misspells it, so don't feel too bad 😉 )

I kind of like the practical example of splitting a CSV, but it does go off the rails a bit towards the end. And `...rest` is out of scope IMO. It might be fine as further reading, but we could also add a note about ignoring the `...rest` section?